### PR TITLE
Use https over http on the site links which ever supports http

### PR DESCRIPTION
**Pull request by prashant-andani**
update site links in the docs to `https` on the site links which ever supports `https`

### CODE REVIEW

File: www/src/components/navigation.js [hunk 1]
```diff
@@ -128,15 +128,6 @@ const Navigation = ({ pathname }) => {
             <NavItem linkTo="/blog/">Blog</NavItem>
             <NavItem linkTo="/showcase/">Showcase</NavItem>
             <NavItem linkTo="/contributing/">Contributing</NavItem>
-            {/* <li css={styles.li}>
```
Comment (KyleAMathews) [inline www/src/components/navigation.js]: You want to put these other changes into another PR?

Comment (KyleAMathews) [reply www/src/components/navigation.js]: It's more difficult to review multiple changes in one PR

Comment (prashant-andani) [reply www/src/components/navigation.js]: raising a separate PR
